src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UIPrefsAccessor.java

```diff
@@ -395,7 +395,7 @@ public String getThemeErrorClass()
    
    public PrefValue<Boolean> useDataImport()
    {
-      return bool("use_dataimport", false);
+      return bool("use_dataimport", true);
```

Comment:
Although since no one is running v0.99-1000 this isn't likely to be a problem!!!